app/models/application_work_experience.rb

```diff
@@ -9,4 +9,22 @@ class ApplicationWorkExperience < ApplicationExperience
   }
 
   audited associated_with: :application_form
+
+  def next_work_breaks_in_months
```

Comment:
Can we extract this from the model? I also suspect it [duplicates some of CheckBreaksInWorkHistory](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/app/services/check_breaks_in_work_history.rb), can you check?

Reply:
Basically, what `CheckBreaksInWorkHistory` does (I think we need a better name for it)
is to find out if there are breaks between works, just true/false.
What now we need is the entire list of gaps in months.
I am going to extract these methods here in a new service that returns the list of work breaks, at that point `CheckBreaksInWorkHistory` will be redundant.